fabric/build.gradle

```diff
@@ -82,7 +84,7 @@ curseforge {
         id = curseforge_id
         releaseType = 'release'
         addGameVersion "Fabric"
-        addGameVersion curseforge_mc_version
```

Comment:
We can't do this because sometimes the curseforge minecraft version mismatches with the actual minecraft version.